ext/bcmath/libbcmath/src/bcmath.h

```diff
@@ -78,6 +78,66 @@ typedef struct bc_struct {
 #define LONG_MAX 0x7ffffff
 #endif
 
+/* This will be 0x01010101 for 32-bit and 0x0101010101010101 for 64-bit */
```

Comment:
Perhaps this swar and byte magic stuff should be in a separate header, since it's really internal stuff. I'm not entirely sure though.

Reply:
Is it ok to duplicate this in each file, or should I have a separate header file?

Reply:
I think a new internal BC Math header is better, such headers exist for various extensions.

Reply:
Thx, do you have a good name idea?

my ideas:
- bcmath_utils.h
- bcmath_ex.h
- internal_bcmath.h

Reply:
The PDO drivers and GMP use the `*_int.h` convention, Phar and ext/com use the `*_internal.h` convention, MySQLi and MySQLnd use the `*_priv.h` convention, and ext/filter and ext/curl use the `*_private.h` convention.

I think thus either using `bcmath_internal.h` or `bcmath_private.h` for the name is the best idea.

Reply:
I got it.
I just noticed that `private.h` already exists in bcmath, is it reasonable to rename it to `bcmath_private.h` and use this? Or do you think it's better to create a new `bcmath_internal.h` since that is just for libbcmath?

Reply:
Aren't these new routines just used within libbcmath? In which case reusing the existing `private.h` header makes sense to me

Reply:
Okay, thx!

Reply:
I moved some constants and macros to `private.h`